GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -1533,10 +1632,18 @@ - (void)OAuthLoginWithAddScopesFlow:(BOOL)addScopesFlow
                                        hint:_hint
                            additionalScopes:nil
                                       nonce:nonce
+                                tokenClaims:tokenClaims
                                  completion:completion];
       }
     }
 
+    if (tokenClaimsError) {
+      XCTestExpectation *tokenClaimsErrorExpectation =
```

Comment:
I'm a little confused by this...you make an expectation and then fulfill it right away? Can you help me to understand what is going on here?